lib/ruby-mpd/song.rb

```diff
@@ -24,6 +24,18 @@ def ==(another)
     self.class == another.class && self.file == another.file
   end
 
+  def to_json(*a)
+    {
+      "time" => @time,
+      "file" => @file,
+      "title" => @title,
+      "artist" => @artist,
+      "album" => @album,
+      "albumartist" => @albumartist,
+      "data" => @data
+    }.to_json(*a)
```

Comment:
Are you sure ? `to_json` is mentionned in the official `JSON` [documentation](http://ruby-doc.org/stdlib-2.0.0/libdoc/json/rdoc/JSON.html#module-JSON-label-Generating+JSON)